.gitattributes

```diff
@@ -0,0 +1,5 @@
+# Force LF line endings for text files
+* text=auto eol=lf
```

Comment:
@jorisroovers this line/setting is not related to the PR, but it was in the file I copied and is useful in general. If you want the PR to be atomic, it can be dropped.